tool/benchtool/Plot.py

```diff
@@ -157,6 +157,159 @@ def stacked_barchart_times(
     if show:
         fig.show()
 
+def double_stacked_barchart_times(
```

Comment:
I don't quite recall why we need this code. Why can't we just consider the expected vs. newly computed versions as two different "strategies" and use the existing code for comparing strategies?

Reply:
In fact, I wonder if the easiest thing to do would just be to have the person (in this case, you :)) who is proposing the new workload provide an image of the bar chart they generate. Then there's some code to generate our own bar chart, and the person reviewing the request can just compare the two.